e2e/test/scenarios/dashboard-filters/temporal-unit-parameters.cy.spec.js

```diff
@@ -398,7 +398,7 @@ describe("scenarios > dashboard > temporal unit parameters", () => {
         .should("contain.text", multiBreakoutQuestionDetails.name);
       H.tableInteractive().within(() => {
         cy.findByText("Created At: Year").should("be.visible");
-        cy.findByText("Product → Created At: Week").should("be.visible");
+        cy.findByText("April 24, 2022").should("be.visible");
```

Comment:
Because there is no empty extra stage, auto-pivotting kicks-in.

Before:
<img width="771" alt="Screenshot 2025-01-23 at 19 33 19" src="https://github.com/user-attachments/assets/b36e8aee-9ed4-4830-8d8e-8ac8ebb98f81" />

After:
<img width="675" alt="Screenshot 2025-01-23 at 19 33 57" src="https://github.com/user-attachments/assets/cabe20e1-f98d-4afb-9e77-d8005af0a4d9" />

Reply:
> Because there is no empty extra stage, auto-pivotting kicks-in.

This is how it used to work before #47167 :tada: